Formula/pcre2.rb

```diff
@@ -5,21 +5,8 @@ class Pcre2 < Formula
 
   # Remove `stable` block next release when patches are no longer needed
```

Comment:
Ideally, and as described in this comment; since there are no longer patches needed, the block should be removed.

This of course is likely to also require changes in the livechek rule below that currently depends on this block

Reply:
If there is no stable block the default url will be used for `:stable` so no changes to the livecheck required.